migrate.py

```diff
@@ -1760,6 +1770,8 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-s', '--spec', required=True, dest='spec_dir',
                         help='A directory spec with YAML files that describe how to organize collections')
+    parser.add_argument('--filter', dest='filters', action='append',
```

Comment:
Maybe
```suggestion
    parser.add_argument('-e', '--exclude', dest='filters', action='append',
```

Reply:
Oh, wait, it's vice versa. Right?
```suggestion
    parser.add_argument('-o', '--only', dest='filters', action='append',
```